osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -474,6 +474,7 @@ private void modButtonPressed(Mod selectedMod)
             {
                 if (State.Value == Visibility.Visible) sampleOn?.Play();
                 DeselectTypes(selectedMod.IncompatibleMods, true);
+                if (selectedMod.RequiresConfiguration) ModSettingsContainer.Alpha = 1;
```

Comment:
Simply doing this can cause a bug in which the first click of the customization button would have no effect.

Also strongly recommend testing.

Reply:
I'm probably missing something here, but if I remember correctly customize button executes `ModSettingsContainer.Alpha = ModSettingsContainer.Alpha == 1 ? 0 : 1`, so that shouldn't happen?

Reply:
Seems correct to me on testing.